dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -504,7 +503,7 @@ private Patch diff( HttpServletRequest request )
         return patchService.diff( new PatchParams( mapper.readTree( request.getInputStream() ) ) );
     }
 
-    @RequestMapping( value = "/{uid}/{property}", method = { RequestMethod.PUT, RequestMethod.PATCH } )
```

Comment:
`PUT` for same path is also mapped to collection item operation where it wins (I guess since there it was the only method) so effectively this method never could be called using `PUT` but needed to be called with `PATCH`.